docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -6,15 +6,62 @@ checked: true
 ---
 # Command Line Interface
 
+This topic provides information on how to run TestCafe tests from a command line.
+
+## Installation
```

Comment:
I don't understand why whole installation topic belongs to command-line-interface. I have strong feeling that it shouldn't be here and more likely should implemented as separate topic. For programmatic usage you will have the same steps. Also we need to reference full installation topic from GS